modules/microshift-embed-microshift-build-image.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-offline-deploy.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-microshift-build-image-offline-deployment_{context}"]
+= Building the rpm-ostree image for offline deployments
+
+You can use Image Builder to create `rpm-ostree` system images with embedded {microshift-short} container images. To embed container images, you must add the image references to your Image Builder blueprint.
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the composer-cli tool.
+* You have created and updated an {op-system-ostree} image blueprint for offline use.
+* You have updated the `/etc/osbuild-worker/osbuild-worker.toml` configuration file for offline use.
```

Comment:
Is this file supposed to exist if I have Image Builder and composer-cli tool ? Comment applies to `/etc/osbuild-worker/osbuild-worker.toml` below too.

Reply:
Yes, I had previously listed this as a prereq. @ggiguash?

Reply:
This file is required when running composer-cli, so we only need to mention it in this section.